docs/pages/desktop-access/getting-started.mdx

```diff
@@ -70,7 +72,8 @@ privileges. You can use this to automate installation, uninstallation, and
 certificate updates. For example:
 
 ```code
-$ teleport-windows-auth-setup-v12.0.0-amd64.exe install --cert=teleport.cer -r
+# Installs Teleport Certificate, loads required DLL, disables NLA and reboots
```

Comment:
This new line isn't code, so it doesn't really belong in the code block.

If you feel a description of what the installer does makes sense, that's fine, but let's write it in text form instead of putting it in a code comment.

Reply:
Updated.  The big thing was this does a reboot.